scripts/orka-deploy.sh

```diff
@@ -3,13 +3,26 @@ set -e
 
 CURL_FLAGS='--location --fail'
 
+REQUEST_DATA="\"orka_vm_name\": \"$VM_NAME\",
+    \"gpu_passthrough\": $GPU_PASSTHROUGH"
+
+# Add system_serial if passed
+if [ -n "$SYSTEM_SERIAL" ]; then
+  REQUEST_DATA="$REQUEST_DATA, \"system_serial\": \"$SYSTEM_SERIAL\""
+fi
+
+# Add vm_metadata if passed
+if [ -n "$VM_METADATA" ]; then
+  VM_METADATA_JSON=$(echo "items: ${VM_METADATA}" | yq r -jP -)
```

Comment:
Since the multiline string used here is now formatted like JSON, is there a reason to still use yq here? I.e. can we eliminate the additional dependency and still use jq?

Reply:
yq here helps format the JSON properly, unless jq can do this itself (not familiar with yq/jq) i.e. { "items": [{"key": "firstKey", "value": "firstValue"}] }

Reply:
jq is perfectly capable of formatting JSON ... yq is more or less the equivalent for YAML (and can convert YAML to JSON). I was under the impression that is why we were using yq previously (trying to take input format as YAML and convert it JSON to include in the API request body)

Reply:
awesome, I'll format it with jq and remove unnecessary dependency.  And yeah that was initially why I tried yq